### PR TITLE
feat: Case hotspots

### PR DESCRIPTION
**Pull request by maxwellkrieger**

### CODE REVIEW

**Comment by eldu** [comment]
I'm going to take a look at fixing up the gap chart !

**Comment by maxwellkrieger** [comment]
@eldu to investigate squishification

File: src/views/historical/testing-gap/dashboard.vue
Comment (eldu) [inline src/views/historical/testing-gap/dashboard.vue:1]: Remove from this pr, it's in #223 

remove dashboard and index.vue

**Review by eldu — CHANGES_REQUESTED** [review]
@s-bessey

File: src/views/historical/case-hotspot/index.vue [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+import Dashboard from "@/views/historical/case-hotspot/dashboard.vue";
+import SuspenseComponent from "@/components/base/SuspenseComponent.vue";
+
+import { useQueryParam } from "../../../composables/useQueryParam";
+
+import { fetchKeys } from "../../../utils/firebase";
+import { prettyDate } from "../../../utils/utils";
+
+// const datasetName = "case_hotspots";
```
Comment (eldu) [inline src/views/historical/case-hotspot/index.vue:33]: Ooohhhhhhh commented out code

File: src/utils/utils.ts [hunk 2]
```diff
@@ -60,10 +60,26 @@ export const prettyDate = (date: string): string => {
...
+
+export function formatUsString(usString: number) {
+  return USStringFormat.format(usString);
+}
+
 export const sortByProperty = (property) => (a, b) => {
   let valA = a[property];
   let valB = b[property];
 
+  // if NaN, make MIN_VALUE
```
Comment (eldu) [inline src/utils/utils.ts:76]: Wondering if it should be 0, but I'm also okay with Number.MIN_VALUE !

File: src/views/historical/case-hotspot/index.vue (hunk 1 above)
Comment (s-bessey) [reply src/views/historical/case-hotspot/index.vue:33]: Oh noooo

**Pull request merged**